src/azul/service/hca_response_v5.py

```diff
@@ -143,18 +162,25 @@ def for_bucket(cls, bucket: JSON):
         return bucket['key']
 
 
-class HitEntry(JsonObject):
-    pass
+class HitEntry(AzulJsonObject):
+
+    # By calling the JsonObject constructor with a single positional argument
+    # dictionary instead of keyword arguments we avoid a code path in jsonobject
+    # that performs a deepcopy of the values.
+    # Note: This "trick" cannot be used with a subclass of JsonObject that
```

Comment:
```suggestion
    #
    # Note: This "trick" cannot be used with a subclass of JsonObject that
```

As always, empy line between paragraphs.